configuration/multimedia.md

```diff
@@ -386,3 +384,109 @@ listenAndAnswer(null, null)
 listenAndAnswer("javasound", "sonos:PLAY5:kitchen")
 listenAndAnswer("voskstt", "voicerss", "voicerss:frFR_Axel", "system,rulehli", "javasound", "sonos:PLAY5:kitchen", "fr-FR", "listeningItem")
 ```
+
+### The Built-in Interpreter
+
+These interpreter is available by default and includes a build-it grammar for English, German, French and Spanish that you can extend using the voiceSystem metadata. These are some examples of the built-in English grammar:
+
+```text
+increase the <item name>
+decrease the <item name>
+set the color of the <item name> to red
+put the <item name> to next
+put the <item name> to previous
+play the <item name>
+pause the <item name>
+rewind the <item name>
+fast forward the <item name>
+start the <item name>
+stop the <item name>
+refresh the <item name>
+```
+
+For exact built-in grammar you can refer to the [source code](https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/text/StandardInterpreter.java#L42).
+
+
+#### Target item
+
+The interpreter resolves the item name based on the its label/synonyms and it's parent label/synonyms.
+
+An example of the possible situations will be:
+
+If you have `Group` item labeled as `TV` with a `Dimmer` child `Brightness`.
+The interpreter understands these phrases as the same `turn off tv`, `turn off brightness`, `turn off tv brightness` and `turn off brightness tv`.
+
+If you add a Switch child labeled as `Power` to the group.
+The interpreter understands now also the phrases `turn off power`, `turn off tv power` and `turn off power tv`.
+But the `turn off tv` phase now detect a collision because two matching items accept the `OFF` command.
```

Comment:
but the `turn off tv` phrase now detects a collision because of two matching Items accepting the `OFF` command.